### PR TITLE
Add process/guide - "How to start a github discussion"

### LINKED ISSUE #21: [DOCS] Add process/guide - "How to start a github discussion"
### Changes/Action required
 
 - Add a complete process of **"How to start a github discussion"**, steps will contain a text, screenshot, Video (it can be a screen recording to show the steps), and GIF (if needed)
 
- Path of the file in which **steps** need to be added - [`open-source-with-pradumna/pages/How-to/guide/starting-discussion.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide/starting-discussion.md)

---

If you have any suggestions feel free to Open an [Issue](https://github.com/Pradumnasaraf/open-source-with-pradumna/issues)

#### This addition of the documentation will get added and hyperlinked in [`How-to/README.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/tree/main/pages/How-to) and also get hosted on the website https://Opensource.pradumnasaraf.co

Screenshot from 2022-05-15 14-11-21

### PR DESCRIPTION
**Pull request by KendallDoesCoding**
## 🛠️ Fixes Issue
fixes #21 

## 👨‍💻 Changes proposed
- Add process/guide - "How to start a github discussion"

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
- Hope this works

### CODE REVIEW

**Comment by Pradumnasaraf** [comment]
Hey, @KendallDoesCoding  Thank you for your contribution, but there are some points you need to be understood, 

- You raise the PR before getting assigned to it
- Also try not to comment below every issue, if that is not necessary, other people might get confused.
- Also if you find something first raise the issue.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by Pradumnasaraf** [comment]
Thank you for the PR, some things that need to be fixed and modify
- convert points into steps
- make points simple and concise. 
- Add horizontal screenshot
- Please show in the screenshot how to actually start one of them

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the PR, some things that need to be fixed and modify

- Convert points into steps
- Make points simple and concise.
- Add horizontal screenshot
- Please show in the screenshot how to actually start one of them

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by KendallDoesCoding** [comment]
> Hey, @KendallDoesCoding Thank you for your contribution, but there are some points you need to be understood,
> 
> * You raise the PR before getting assigned to it
> * Also try not to comment below every issue, if that is not necessary, other people might get confused.
> * Also if you find something first raise the issue.

Will follow in future, thanks

**Comment by KendallDoesCoding** [comment]
> * Convert points into steps

By this you mean, instead of the -, replace it to numbering?

**Comment by KendallDoesCoding** [comment]
> Add horizontal screenshot

How can I make the following screenshot, horizontal?:

image

**Comment by KendallDoesCoding** [comment]
@Pradumnasaraf hope this works!

**Comment by Pradumnasaraf** [comment]
Thank you for the changes, but it is still hard to navigate for the beginners, need to add screenshots in every step possible and reduce the text, making it more concise.

Add need to use backtick and bold down some words

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Unreadable

**Comment by Pradumnasaraf** [comment]
> > Add horizontal screenshot
> 
> How can I make the following screenshot, horizontal?:
> 
> image

you can add Full screenshot 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by KendallDoesCoding** [comment]
I tried to make the changes you requested, please review now @Pradumnasaraf

File: pages/How-to/guide/starting-discussion.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
-</div>
+## Starting Discussion in GitHub
+1. Go to the **repo** in which you want to start a discussion in or use this link - **https://github.com/{repo-owner}/{repo-name}/discussions**.
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/starting-discussion.md:2]:
```suggestion
1. Go to the **repo** in which you want to start a discussion or use this link - `https://github.com/{repo-owner}/{repo-name}/discussions`.
```

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to Me!, thank you for your contribution

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by KendallDoesCoding** [comment]
> Looks good to Me!, thank you for your contribution
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good

Thanks, please merge it in!

File: pages/How-to/guide/starting-discussion.md (hunk 1 above)
Comment (Pradumnasaraf) [inline pages/How-to/guide/starting-discussion.md:2]:
```suggestion
1. Go to the **repo** in which you want to start a discussion in or use this link - `https://github.com/{repo-owner}/{repo-name}/discussions`.
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by Pradumnasaraf** [comment]
> > Looks good to Me!, thank you for your contribution
> > __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  
> 
> Thanks, please merge it in!

Sure, just need that small modification

**Comment by KendallDoesCoding** [comment]
> > > Looks good to Me!, thank you for your contribution
> > > __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)
> > 
> > 
> > Thanks, please merge it in!
> 
> Sure, just need that small modification

Done, hope it's good to go!

**Comment by Pradumnasaraf** [comment]
Thank you for the changes, we are good to go!! :rocket: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**